js/packages/openinference-mastra/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@arizeai/openinference-mastra",
+  "version": "1.0.0",
+  "private": false,
+  "type": "module",
+  "types": "dist/esm/index.d.ts",
+  "description": "OpenInference utilities for ingesting Mastra spans",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "tsc --build tsconfig.esm.json && tsc-alias -p tsconfig.esm.json",
+    "postbuild": "echo '{\"type\": \"module\"}' > ./dist/esm/package.json && rimraf dist/test",
+    "type:check": "tsc --noEmit",
+    "test": "vitest"
```

Comment:
can you file a ticket to move all js stuff to vitest and work with Donny to get this moved over?

Reply:
https://github.com/Arize-ai/openinference/issues/1652